docs/release_notes/8.4.md

```diff
@@ -96,7 +96,7 @@ As with any release, there are sometimes bugs or issues found in the process of
 
 !!! note "A Note about Secure Boot"
     We know many of you depend on Secure Boot. It is a non-trivial process to get Secure Boot for a new OS. This process is underway and the [shim-review](https://github.com/rhboot/shim-review) process should begin very soon.
-    Rocky Linux 8.4 will initially be released without Secure Boot support enabled, however once the proper packages have been built and signed, a second set of ISOs will be released for the Rocky Linux 8.4 with Secure Boot support.
+    Rocky Linux 8.4 will initially be released without Secure Boot support enabled, however, once the proper packages have been built and signed, a second set of ISOs will be released for the Rocky Linux 8.4 with Secure Boot support.
```

Comment:
I got some feedback that the wording was confusing, feel free to reword if you'd like!

Reply:
I understand and agree with your statements, I do believe the documentation above should be reworded to this though. Seems to be easier to understand and more specific: 

Reworded:
> Rocky Linux version 8.4 will initially be released **without** Secure Boot support enabled by default. However, once the proper packages have been built and signed, another set of ISOs for Rocky Linux version 8.4 will be released **with** Secure Boot support available.